.dockerignore

```diff
@@ -0,0 +1,18 @@
+Live Share/
+.git/
+.vagrant/
+todo_app/__pycache__/
+/todo_app.egg-info/
+data/
+Vagrantfile*
+todo_app/test sections.py
+requirements.txt
+.gitignore
+Original_Vagrant_File
+Dockerfile
+Dockerfile2
+To-do-GU1.dockerfile
+todo_app/__init__.py
+todo_app/gunicorn_config.py
+.env
+.env.template
```

Comment:
This looks sensible, but it's worth noting that this doesn't exclude the `Trello_API_Keys.txt` currently - that makes sense since your app needs that but we want to avoid baking any secrets into our docker images. Changing the system to use the `.env` way of loading environment variables will allow us to avoid storing the secrets in the image